src/components/LocationSearch.js

```diff
@@ -1,12 +1,44 @@
 import { useState } from 'react'
 import FormControl from '@mui/material/FormControl'
 import TextField from '@mui/material/TextField'
+import { Autocomplete} from '@react-google-maps/api';
 
-const LocationSearch = ({type, initialLocation, onLocationChange}) => {
+const LocationSearch = ({type, initialLocation, onLocationChange,initialLocationChange,sampleText}) => {
     const [location, setLocation] = useState(initialLocation)
+    console.log(process.env.REACT_APP_API_KEY);
+  const [autocomplete,setAutoComplete]=useState(null);
+  
+  const onLoad =(autocomplete)=> {
+    console.log('autocomplete: ', autocomplete)
+
+    setAutoComplete(autocomplete);
+  }
+
+  const onPlaceChanged= ()=> {
+    if (autocomplete !== null) {
+      let place=autocomplete.getPlace();
+      console.log(place) //result of the selected location from dropdown
+      console.log("lat",place.geometry['location'].lat());
+      console.log("lng",place.geometry['location'].lng());
+      let locationObj={};
+      locationObj.display=place?.name;
+      locationObj.latLong=place?.geometry?.location.lat()+","+place?.geometry?.location.lng();
+      console.log(locationObj);
+      setLocation(locationObj);
+      initialLocationChange(locationObj);
+    }
+  }
     return (
+        
```

Comment:
Did it work? You cant have 2 root elements in render

Reply:
<img width="674" alt="Screenshot 2022-12-19 at 4 01 44 PM" src="https://user-images.githubusercontent.com/56752427/208405801-555b1e61-36f1-4fce-b8e7-d0b124557b02.png">

There is only 1 root element FormControl